docs/logaxis.md

```diff
@@ -0,0 +1,27 @@
+## jquery.flot.logaxis
+This plugin is used to create logarithmic axis. This includes tick generation,
+formatters and transformers to and from logarithmic representation.
+
+### Methods and hooks
```

Comment:
I think you should specify that these methods are mostly for testing and not for public usage.

Reply:
Actually they are hooks (like setRange hook) or transformers or formatters that are overriding the default ones